packages/velo-external-db/test/storage/schema_provider.spec.js

```diff
@@ -85,16 +86,19 @@ describe('Schema API', () => {
             await expect(env.schemaProvider.addColumn(ctx.collectionName, { name: ctx.columnName, type: 'datetime', subtype: 'timestamp' })).rejects.toThrow(CollectionDoesNotExists)
         })
 
-        if (shouldNotRunOn(['Google-Sheet'], name)) {
-            test('add column on a an existing collection', async() => {
-                await env.schemaProvider.create(ctx.collectionName, [])
-
-                await env.schemaProvider.addColumn(ctx.collectionName, { name: ctx.columnName, type: 'datetime', subtype: 'timestamp' })
 
-                await expect( env.schemaProvider.describeCollection(ctx.collectionName) ).resolves.toEqual( hasSameSchemaFieldsLike([{ field: ctx.columnName, type: 'datetime' }]))
-            })
-        }
+        test('add column on a an existing collection', async() => {
+            const neededOperations = [ADD_COLUMN]
 
+            if ( neededOperations.every(op => env.schemaOperations.includes(op)) ) {
+                await env.schemaProvider.create(ctx.collectionName, [])
+                await env.schemaProvider.addColumn(ctx.collectionName, { name: ctx.columnName, type: 'datetime', subtype: 'timestamp' })
+                await expect( env.schemaProvider.describeCollection(ctx.collectionName) ).resolves.toEqual( hasSameSchemaFieldsLike([{ field: ctx.columnName  }]))
+            }
+            else {
+                expect(true).toBe(true)
```

Comment:
Maybe export this to a function as well. 
`passTest` or something similar that includes that assertion.